packages/api-client/src/views/Request/RequestSubpageHeader.vue

```diff
@@ -43,16 +40,12 @@ const { currentRoute } = useRouter()
         ]"
         :modelValue="modelValue"
         @update:modelValue="$emit('update:modelValue', $event)" />
-      <!-- todo: only show this env selectorin modal -->
-      <EnvironmentSelector
```

Comment:
@cameronrohani this needed to be removed right? Currently it isn't shown at all

Reply:
we are planning on showing env selector when it's ready for the modal, so you can remove it for now but I believe the plan is to add it back